logic/src/main/nl/jft/logic/match/Goal.java

```diff
@@ -0,0 +1,8 @@
+package nl.jft.logic.match;
+
+/**
+ * @author Lesley
+ */
+public class Goal {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this empty class, write its code or make it an "interface". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://92.110.24.119:9000/coding_rules#rule_key=squid%3AS2094)